strapi/strapi-plugin-seo
# Convert SEO plugin to strapi v5

### What does it do?

- Converts the SEO plugin to strapi v5

### Related issue(s)/PR(s)

https://strapi-inc.atlassian.net/browse/DX-1428	

Uses additional exports added to the monorepo to be tested with https://github.com/strapi/strapi/pull/21209 if not merged

To test:
- Clone this branch
- Run this in the plugin codebase `yarn; yarn run watch:link` - This will publish the package to yalc locally. (make sure to install yalc globally https://www.npmjs.com/package/yalc)
- In a strapi v5 app run `yarn dlx yalc add --link @strapi/plugin-seo && yarn install`. This will install the plugin as a dependency from yalc


DX-1428
